src/threads_and_async.md

```diff
@@ -75,38 +75,35 @@ You can even use commands for synchronous operations too.
+By default Relm4 uses only one thread for asynchronous background tasks which might not be enough. For example, if you spawn many commands in your application, you can set `RELM_THREADS` to 4 to increase the thread count by 3 additional threads.
 
 > Note: Setting the static variables must be done early.
 > As soon as the runtime is initialized (which happens when it's accessed for the first time), the values cannot be changes anymore.
 
 ## Workers
 
 Workers are basically just components without widgets.
-However, because they don't have widgets, workers can be sent to other threads which also makes it possible run their update function on another thread.
-This means, that running the slow `generate_rsa_key()` function from the first section won't freeze your application.
+However, since they don't have widgets, workers can be sent to other threads, which also makes it possible run their update function on another thread.
+This means, that executing the slow `generate_rsa_key()` function from the first section won't freeze your application.
 
-All you need to do to create a worker is to implement the [`Worker`](https://relm4.org/docs/next/relm4/worker/trait.Worker.html) trait and call [`detach_worker`](https://relm4.org/docs/next/relm4/component/struct.ComponentBuilder.html#method.detach_worker) from the `ComponentBuilder`.
-This will setup the worker to run on a new thread.
+To create a worker, you just need to implement the [`Worker`](https://relm4.org/docs/next/relm4/worker/trait.Worker.html) trait and call [`detach_worker`](https://relm4.org/docs/next/relm4/component/struct.ComponentBuilder.html#method.detach_worker) from the `ComponentBuilder`. This will execute the worker in a new thread.
 
 You might wonder why workers even exist if we already have commands.
-Actually, workers have some unique properties: They can only run one task at the time and store state like a regular component.
+Actually, workers have some unique properties: They can only run one task at a time and store their state like a regular component.
 
 ## Local futures
 
 Both commands and workers run on different threads, so Rust's guarantees require the involved types to implement `Send`.
-This can be problematic sometimes, for example when widgets are involved.
+This can sometimes be problematic, for example when it comes to widgets.
 
 Fortunately, the [`spawn_local`](https://relm4.org/docs/next/relm4/fn.spawn_local.html) function allows us to spawn local futures, which don't require `Send` because they run on the main thread.
 This works because GTK uses an event loop from GLib to handle asynchronous events, which also allows you to execute futures.
 
-The only downside of this solution is that not all async libraries are fully compatible with the GLib executor because they have to use Tokio.
+The only drawback of this solution is that not all async libraries are fully compatible with the GLib executor, since they must use Tokyo.
```

Comment:
Maybe it's better to say "because they depend on tokio" (Tokyo is definitely wrong though)